_internals/software-architecture.md

```diff
@@ -5,10 +5,11 @@ nav_order: 1
 use_mermaid: true
 ---
 
-Babelfish implements Microsoft SQL Server behavior on top of PostgreSQL. To achieve this goal, a
-couple of areas had to be modified inside the PostgreSQL core. PostgreSQL does
-not offer this capability out-of-the-box, therefore add-ons had to be provided
-to make this happen.
+Babelfish implements Microsoft SQL Server behavior on top of PostgreSQL.
```

Comment:
Suggested simplification for lines 9 - 12:

The server behavior is modified primarily through the use of [extensions](https://www.postgresql.org/docs/current/extend-extensions.html),
keeping modifications to the actual PostgreSQL server at a minimum.

Reply:
I have simplified the wording in 45199178b8963e0c1cbcd8eab4b0a08a319240be.
I went with "PostgreSQL core", since an extension can also be seen as a modification.